recipes/meta/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -ex
```

 then you don't need to `|| exit 1` at all time.